src/ota_proxy/cache_streaming.py

```diff
@@ -82,7 +75,10 @@ class CacheTracker(Generic[_WEAKREF]):
             finish the caching.
     """
 
-    READER_SUBSCRIBE_WAIT_PROVIDER_TIMEOUT = 2
+    SUBSCRIBER_WAIT_PROVIDER_READY_MAX_RETRY = 16  # 9s
+    SUBSCRIBER_WAIT_NEXT_CHUNK_MAX_RETRY = 16  # 9s
+    SUBSCRIBER_WAIT_BACKOFF_FACTOR = 0.01
+    SUBSCRIBER_WAIT_BACKOFF_MAX = 1
```

Comment:
Increase the timeout a lot here considering low speed/unstable network environment.